docs/notebooks/WMD_tutorial.ipynb

```diff
@@ -0,0 +1,603 @@
+    "\n",
+    "The reason it is beneficial to use normalized vectors is that word2vec is optimized for cosine distance (see [cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity)), which measures the angle between vectors, but WMD uses Euclidean distance. The Euclidean distance between two vectors might be large because their lengths differ, but the cosine distance is small because the angle between them is small; we can mitigate some of this by normalizing the vectors.\n",
+    "\n",
+    "Note that normalizing the vectors can take some time, especially if you have a large vocabulary and/or large vectors.\n",
+    "\n",
+    "Usage is illustrated in the example below. It just so happens that the vectors we have downloaded are already normalized, so it won't do any difference in this case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cell took 12.25 seconds to run.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Normalizing word2vec vectors.\n",
+    "start = time()\n",
+    "\n",
+    "model.init_sims(replace=True)  # Normalizes the vectors in the word2vec class.\n",
+    "\n",
+    "distance = model.wmdistance(sentence1, sentence2)  # Compute WMD as normal.\n",
+    "\n",
+    "print 'Cell took %.2f seconds to run.' %(time() - start)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Similarity queries using `WmdSimilarity`\n",
+    "\n",
+    "You can use WMD to get the most similar documents to a query, using the `WmdSimilarity` class. Its interface is similar to what is described in the [Similarity Queries](https://radimrehurek.com/gensim/tut3.html) Gensim tutorial.\n",
+    "\n",
+    "> **Important note:**\n",
+    ">\n",
+    "> WMD is a measure of *distance*. The similarities in `WmdSimilarity` are simply the *negative distance*. Be careful not to confuse distances and similarities.\n",
+    "\n",
+    "### Yelp data\n",
+    "\n",
+    "Let's try similarity queries using some real world data. For that we'll be using Yelp reviews, available at http://www.yelp.com/dataset_challenge. Specifically, we will be using reviews of a single restaurant.\n",
+    "\n",
+    "This time around, we are going to train the Word2Vec embeddings on the data ourselves. One restaurant is not enough to train Word2Vec properly, so we use 6 restaurants for that, but only run queries against one of them.\n",
+    "\n",
+    "The restaurants we chose were those with the highest number of reviews in the Yelp dataset. The corpus we trained Word2Vec on has 18957 documents (reviews), and the corpus we used for `WmdSimilarity` has 4137 documents.\n",
```

Comment:
What are the names and locations of these restaurants? What is 4bEjOyTaDG24SY5TxsaUNQ ? Why did you choose it?